dohproxy/httpproxy.py

```diff
@@ -41,6 +42,9 @@ def parse_args(args=None):
             status=415, body=b'Unsupported content type'
         )
 
+    # Put client IP address in the request
```

Comment:
I don't understand this. Isn't the point of the middleware to set request.remote accordingly?